tasmota/tasmota_xdrv_driver/xdrv_01_9_webserver.ino

```diff
@@ -2594,7 +2591,7 @@ void HandleWifiConfiguration(void) {
                   for (uint32_t k = 0; k < 4; ++k) {
                     WSContentSend_P(PSTR("<i class='b%d%s'></i>"), k, (num_bars < k) ? PSTR(" o30") : PSTR(""));
                   }
-                  WSContentSend_P(PSTR("</span></div></div>"));
+                  WSContentSend_P(PSTR("</div></span></div>"));
```

Comment:
Wrong. The original is correct

Reply:
I'd a look at the resulting HTML code.

This is the code before the PR:
```html
1:	<div title='-25 dBm (100%)'>
2:		<a href='#p' onclick='c(this)'>SSID</a>
3:		<span class='q'>
4:			<div class='si'>
5:				<i class='b0'></i><i class='b1'></i><i class='b2'></i><i class='b3'></i>
6:			</span>
7:		</div>
8:	</div>
```
We see opening DIV-SPAN-DIV and closing SPAN-DIV-DIV. The lines 6 and 7 have to be swapped.

After the PR it looks like this and the order is correct:
```html
1:	<div title='-25 dBm (100%)'>
2:		<a href='#p' onclick='c(this)'>SSID</a>
3:		<span class='q'>
4:			<div class='si'>
5:					<i class='b0'></i><i class='b1'></i><i class='b2'></i><i class='b3'></i>
6:			</div>
7:		</span>
8:	</div>
```

Reply:
You're correct.